style.css

```diff
@@ -0,0 +1,499 @@
+}
+
+.icons-container {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.icons-containers {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  justify-items: center;
+  position: relative;
+  right: 63px;
+  bottom: 80px;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+.icons-containers img {
+  width: 20px;
+  height: 20px;
+}
+
+.second_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  justify-items: center;
+  margin-top: 90px;
+}
+
+.second_container h2 {
+  color: #1c1a19;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 35px;
+}
+
+.line {
+  border: 1px solid transparent;
+  background-color: black;
+  opacity: 0.6;
+  width: 270px;
+  height: 0.5px;
+}
+
+.header_title {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  justify-items: center;
+  gap: 20px;
+}
+
+.recent_work_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  margin-top: 30px;
+  align-items: center;
+}
+
+.recent_work_container img {
+  width: 280px;
+}
+
+.recent_work_container_text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  margin-left: 20px;
+  margin-top: 10px;
+  gap: 10px;
+}
+
+.recent_work_container_text h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.recent_work_container_text p {
+  width: 300px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+}
+
+.butons_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 10px;
+}
+
+.butons_container button {
+  text-align: center;
+  background: none;
+  height: 40px;
+  width: 60px;
+  border: 1px solid rgb(61, 59, 59);
+  position: relative;
+  left: -17px;
+  background-color: aliceblue;
+}
+
+.recent_work_container .project_btn {
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  font-weight: bold;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: repeat(6, 1fr);
+  margin-top: 20px;
+  overflow: hidden;
+}
+
+.box {
+  padding: 23px;
+  background-image: url('./images/placeholder.png');
+  width: 240px;
+  height: 300px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+}
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), kindly use Flexbox to lay out individual cards. 👍